.travis.yml

```diff
@@ -24,7 +24,7 @@ matrix:
 install: travis_retry composer update --no-interaction $COMPOSER_FLAGS
 
 script:
- - vendor/bin/phpunit --verbose --coverage-clover=coverage.xml
+ - phpunit --verbose --coverage-clover=coverage.xml
```

Comment:
I think you are currently missing the install step for PHPUnit, but moving it to the makefile would also solve that issue IMO.

Reply:
We're using the system-installed PHPUnit on CI. I'm pretty sure there's no one version of PHPUnit that can work on all the versions of PHP we currently test, so using composer bin there is a bit problematic.

Unfortunately, the Travis system-installed PHPUnit is outdated on PHP 5.5, apparently, so the build is failing.